utils/wasmtime.hh

```diff
@@ -0,0 +1,2665 @@
+/**
+ * Downloaded from https://github.com/bytecodealliance/wasmtime-cpp/blob/main/include/wasmtime.hh
+ * License: Apache License 2.0
+ */
+
```

Comment:
We need this in the licenses/ directory too.

Reply:
We already do: https://github.com/scylladb/scylla/blob/master/licenses/apache-license-2.0.txt

Reply:
Ah, then /NOTICE.txt is the place to mention imported code.